readme.md

```diff
@@ -1,74 +1,55 @@
+  - `api_url`: depends on the server that you request, e.g. https://app.datasaur.ai. This one should be adjusted if you're on a self hosted version.
+  - `client_id` and `client_secret`: generated from OAuth credentials, see [here](https://datasaurai.gitbook.io/datasaur/advanced/apis-docs/oauth-2.0).
 
-## Get Projects
+### Create Project
 
-Get projects query returns a paginated response. The input variables for the pagination can be configured directly in `get_projects.json`.
-Refer to https://api-docs.datasaur.ai/#definition-GetProjectsFilterInput for available filters.
+- The detailed explanation can be accessed [here](https://datasaurai.gitbook.io/datasaur/advanced/apis-docs/create-new-project).
+- Note that the process is asynchronous.
+- This is a multipart request because it needs to handle a list of files that will be used when creating the project.
+- Reference: `/create-project-async` and follow the README.
 
-```
-python get_projects.py https://app.datasaur.ai <client_id> <client_secret>
-```
+### Export Project
 
-## Update Project Tags
+- The detailed explanation can be accessed [here](https://datasaurai.gitbook.io/datasaur/advanced/apis-docs/export-project).
+- Syntax: `python3 export.py <api_url> <client_id> <client_secret> <project_id> <filename> <export_format> <output_dir>`
+  - `project_id`: project resource ID that can be accessed from the URL, e.g. `YOfkM6jKHzN` will be the ID if the project URL is https://app.datasaur.ai/teams/1/projects/YOfkM6jKHzN/review/ac35a379-2367-4d25-81d5-cf1184832b30.
+  - `filename`: any alphanumeric string that will be used as the export result name (without the extension).
+  - `export_format`: the list of value options could be seen on the GitBook above.
+  - `output_dir`: prefix that will be added to specify the export result path.
 
-Tag names and project id can be specified from update_project_tags.json.
+### Get Projects
 
-Two available methods: PUT and PATCH. PUT method will replace all of the project tags with the input, just like PUT method on REST API. PATCH method will only add new tags to a project. See the example below.
+- The detailed explanation can be accessed [here](https://datasaurai.gitbook.io/datasaur/advanced/apis-docs/get-list-of-projects).
+- This query returns a paginated response.
+- The input variables for the pagination can be configured directly on `get_projects.json`.
+- Syntax: `python3 get_projects.py <api_url> <client_id> <client_secret>`
 
-- Project A has Tag1.
-- PUT ["Tag2"]: Project A will have only Tag2.
-- PATCH ["Tag2"]: Project A will have Tag1 and Tag2.
+### Update Project Tags
 
-```
-python update_project_tags.py https://app.datasaur.ai <client_id> <client_secret> <team_id> <method>
-python update_project_tags.py https://app.datasaur.ai <client_id> <client_secret> <team_id> PUT
-```
+- Tag names and project ID can be specified from `update_project_tags.json`.
+- Two available methods: PUT and PATCH.
```

Comment:
eh, do we have these methods?

🤔

Reply:
the methods are introduced when creating the update project tags script based on Robosaur capability
the API doesn't have the functionality though

Reply:
ow right got it